src/app/(main)/graduation-progress/layout.module.scss

```
@@ -1,12 +1,14 @@
 @use '@/styles/utilities' as utilities;
 @use '@/styles/color.scss' as color;
+@use '@/styles/spacing.scss' as spacing;
 
 .container {
-  @include utilities.full-height-layout;
-  background-color: color.$gray-300;
+  @include utilities.app-layout;
+  background-color: color.$white;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)